DEVELOPERS.md

```diff
@@ -0,0 +1,214 @@
+# Developing for Accord Project
+
+* [Development Setup][developers.setup]
+* [Coding Rules][developers.rules]
+* [Commit Message Guidelines][developers.commits]
+* [Writing Documentation][developers.documentation]
+
+## <a name="setup"></a> Development Setup 
+
+This document describes how to set up your development environment to build and test Cicero-UI, and
```

Comment:
There are still references to `Cicero-UI`, but these need to be changed to `Accord Project Models` and adapted accordingly.